trunk/NDHMS/.nwm_version

```diff
@@ -0,0 +1 @@
+v1.2.1+dev
```

Comment:
this should be blank?

Reply:
yeah, probably (until we create that branch?)